Binary files www/doc//check.info and check-0.9.8/doc//check.info differ
diff -dub www/doc//check.texi check-0.9.8/doc//check.texi
--- www/doc//check.texi	2009-09-23 14:44:00.000000000 +0400
+++ check-0.9.8/doc//check.texi	2011-08-18 02:04:36.000000000 +0400
@@ -8,6 +8,11 @@
 @syncodeindex vr cp
 @c %**end of header
 
+@dircategory Development
+@direntry
+* Check: (check)Introduction.
+@end direntry
+
 @copying
 This manual is for Check
 (version @value{VERSION}, @value{UPDATED}),
@@ -26,11 +31,6 @@
 @end quotation
 @end copying
 
-@dircategory Software development
-@direntry
-* Check: (check)Introduction.
-@end direntry
-
 @titlepage
 @title Check
 @subtitle A Unit Testing Framework for C
Common subdirectories: www/doc//example and check-0.9.8/doc//example
